roles/kubernetes/preinstall/tasks/0081-ntp-configurations.yml

```diff
@@ -74,8 +74,33 @@
     - not is_fedora_coreos
     - not ansible_os_family in ["Flatcar", "Flatcar Container Linux by Kinvolk"]
 
-- name: Set timezone
+- name: Gather selinux facts
+  ansible.builtin.setup:
+    gather_subset: selinux
+  when:
+    - ntp_timezone
+    - ansible_os_family == "RedHat"
+
+- name: Put SELinux in permissive mode, logging actions that would be blocked.
+  ansible.posix.selinux:
+    policy: targeted
+    state: permissive
+  when:
+    - ntp_timezone
+    - ansible_os_family == "RedHat"
+    - ansible_facts.selinux.mode == 'enforcing'
+
+- name: Set ntp_timezone
   community.general.timezone:
     name: "{{ ntp_timezone }}"
   when:
     - ntp_timezone
+
+- name: Re-enable SELinux
+  ansible.posix.selinux:
+    policy: targeted
+    state: enforcing
+  when:
+    - ntp_timezone
+    - ansible_os_family == "RedHat"
+    - ansible_facts.selinux.status == 'enabled'
```

Comment:
It seems the description above is for a temporary switch, but what I'm more confused about is why `enabled` has to be set to '`enforcing`.

Reply:
See line #91 SELinux only needs to be toggled when it is, initially, enforcing.

Reply:
But if the initial setting is permissive, would the result be `enforcing`?